assets/css/typography.css

```diff
@@ -1,4 +1,31 @@
 @layer base {
+  /* Roboto Flex */
+  @font-face {
```

Comment:
It's worth including a comment where this content originated from. i.e., if this CSS was generated by Google's API, we can include a comment that describes the URL to use to get this content (in case we want to / need to update / refresh this in future).


☝️ Same can apply to the downloaded fonts. We could add a README (or otherwise) in that directory to document how to obtain / refresh them (for "future self" - these things tend to be forgotten, and now you need to dig "what options did we use for these??")

Reply:
Good point about "future self". Let me add a comment here.

Reply:
ChatGPT helped me write the font faces, so I think attribution there will be difficult :P